osbs/api.py

```diff
@@ -170,24 +171,34 @@ def get_pod_for_build(self, build_id):
 
     @osbsapi
     def get_build_request(self, build_type=None, inner_template=None,
-                          outer_template=None, customize_conf=None):
+                          outer_template=None, customize_conf=None,
+                          arrangement_version=5):
```

Comment:
Should this be `DEFAULT_ARRANGEMENT_VERSION`?

Reply:
Fixed.